core/components/footer/footer.tsx

```diff
@@ -70,29 +87,12 @@ export const Footer = async () => {
   ];
 
   return (
-    <ComponentsFooter
-      contactInformation={data.settings?.contact ?? undefined}
-      copyright={
-        data.settings
-          ? `© ${new Date().getFullYear()} ${data.settings.storeName} – Powered by BigCommerce`
-          : undefined
-      }
```

Comment:
We seem to be hardcoding "SOUL" here instead of using the store name. We should preserve the functionality that was there in `main` as much as possible while making it fit with the VIBE